roles/core/dhcp_server/readme.rst

```diff
@@ -75,7 +75,7 @@ Mandatory inventory vars:
    * .services_ip.pxe_ip
    * .services_ip.dns_ip
    * .services_ip.time_ip
-* dhcp_settings
+* dhcp_server_settings
    * .default_lease_time
    * .max_lease_time
```

Comment:
The new variable names are `dhcp_server_default_lease_time` and `dhcp_server_max_lease_time`.

Reply:
You are right; I also moved them into optional.